int64 input
---
sensor_msgs/Image output
